skimage/measure/block.py

```diff
@@ -59,7 +60,9 @@ def block_reduce(image, block_size, func=np.sum, cval=0, func_kwargs=None):
             [35]]])
     """
 
-    if len(block_size) != image.ndim:
+    if np.isscalar(block_size):
+        block_size = (block_size,) * image.ndim
```

Comment:
This LOC needs a unit test. Thanks.

Reply:
I added a unit test here. 
https://github.com/scikit-image/scikit-image/blob/1b501b6518e93c7a52ecee13507d60d7525b8b99/skimage/measure/tests/test_block.py#L98-L106